From 5f618cec83012d97408674895392c08cf557cfc2 Mon Sep 17 00:00:00 2001
From: cs469 <cs469@cs469.cs.toronto.edu>
Date: Fri, 4 Oct 2019 18:04:03 -0400
Subject: [PATCH] done

---
 init/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/main.c b/init/main.c
index 96f8d5af52d6..77537cb82324 100644
--- a/init/main.c
+++ b/init/main.c
@@ -592,6 +592,7 @@ asmlinkage __visible void __init start_kernel(void)
 	 */
 	boot_cpu_init();
 	page_address_init();
+	pr_notice("Welcome to Matthew Brousseau - brouss16's kernel!");
 	pr_notice("%s", linux_banner);
 	setup_arch(&command_line);
 	mm_init_cpumask(&init_mm);
-- 
2.20.1

